docs/development_suite/identity-and-access-management/user-management.md

```diff
@@ -107,21 +105,55 @@ The Company Owner can add a new User to a Company by pressing the *Add user* but
+
+Now let us remember, however, that one of those Developers (the first one) was initially assigned the `Developer` role at individual level. This means that one Developer will have both an individual role and a role conferred by the group membership. You don’t have to worry about this, as roles in Console always add up and therefore it does not represent a problem of any kind.
+
+The dual management of users – through a combination of individual roles and group roles – also allows for more personalized governance of access to the resources of your Company: imagine, for example, that the initial Junior Developer (who has both an individual role and the role attributed by group membership) has now become a Developer Expert, having increased the responsibilities within some of the Projects of your Company. At that point, you might want to give that user additional privileges only on those specific Projects. You can do it by assigning to that user at Project level a role that expands the capabilities currently enabled by the Developer role assignment at Company level (e.g., being able to deploy on a specific Projects by assigning the `Maintainer` role to that user).
+
+Nevertheless, once again, managing users through groups can be useful when the number of users to which to assign roles on specific resources increases. As a matter of fact, thanks to groups, you can actually increase the capabilities on specific resources for a batch of users in your Company in an easier way.
+
+More practically speaking, if you want to extend privileges to all your Junior developers on a specific Project (or Environment of your project) (e.g., by elevating that group's role to `Maintainer`, in order to provide them with Deploy trigger permission), with just one action you can extend that privilege to all of your Junior developers: in this case, you only have to assign to the "My Company Developers" group the `Maintainer` role specifically on the desired Project/Runtime Environment.
+
+Otherwise, if they were not managed through groups, this action of assigning `Maintainer` role on that Project (or Project Runtime Environment) would have to be repeated for each of the members, which is – again – a time-consuming (and even error-prone) activity. Moreover, if this extension of privileges was meant to be granted just for a limited period of time, then the subsequent activity of role disassignment for each of those members highlights this criticality even more.
+
+By contrast, through groups, for instance the `Maintainer` role disassignment from that specific Project (or Project Runtime Environment) by granting the `Developer` role assigned at Company level would have been much more effective and circumscribed activity, keeping access governance on that Project more organized and clear.
```

Comment:
```suggestion
Otherwise, if they were not managed through groups, this action of assigning `Maintainer` role on that Project (or Project Runtime Environment) would have to be repeated for each of the members, which is – again – a time-consuming (and even error-prone) activity.  
Moreover, if you wanted to grant this privilege just for a limited period of time, then restoring the initial role setting would have been done manually once again for each of those members, highlighting this criticality even more.

By contrast, through groups, for instance the `Maintainer` role disassignment from that specific Project (or Project Runtime Environment) by granting the `Developer` role assigned at Company level would have been much more effective and circumscribed activity, keeping access governance on that Project more organized and clear.
```